src/views/ProductInventory.vue

```diff
@@ -760,7 +757,7 @@ export default defineComponent({
   align-items: center;
 }
 
-.open-orders > ion-card > ion-item > ion-label:not(:first-child) {
+.orders:last-child > ion-card > ion-item > ion-label:not(:first-child) {
```

Comment:
it is better to use a class here since the card isn't placed where it is out of semantics, its just out of happenstance. So a class selector is a better choice.

Reply:
I have applied ionic utility class to center text of ion-label.
https://ionicframework.com/docs/layout/css-utilities#:~:text=class%3D%22ion%2Dtext%2Dcenter%22